.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,4 +2,7 @@ Before submitting a pull request, please take a look at our
 [Contributing](https://github.com/getsentry/sentry-javascript/blob/master/CONTRIBUTING.md) guidelines and verify:
 
 - [ ] If you've added code that should be tested, please add tests.
-- [ ] Ensure your code lints and the test suite passes (`yarn lint`) & (`yarn test`).
+- [ ] Ensure your code lints and the test suite passes (`yarn build && yarn lint && yarn test`).
+
+PS: remember to install the following extension on VSCode to run the tests:
+https://marketplace.visualstudio.com/items?itemName=augustocdias.tasks-shell-input
```

Comment:
I don't see much reason to tell people to download this extension. Running the yarn commands above in the terminal is probably easier.

Reply:
Agreed. I realised it was already part of the recommended extensions in `.vscode`. I'll remove this comment.
The main modification in this file is adding the `yarn build`, which took me some time to understand.